ddtrace/contrib/pytest/plugin.py

```diff
@@ -123,8 +124,14 @@ def pytest_runtest_protocol(item, nextitem):
         # Parameterized test cases will have a `callspec` attribute attached to the pytest Item object.
         # Pytest docs: https://docs.pytest.org/en/6.2.x/reference.html#pytest.Function
         if getattr(item, "callspec", None):
-            params = {"arguments": item.callspec.params, "metadata": {}}
-            span.set_tag(test.PARAMETERS, json.dumps(params, default=repr))
+            parameters = {"arguments": {}, "metadata": {}}  # type: Dict[str, Dict[str, str]]
+            for param_name, param_val in item.callspec.params.items():
```

Comment:
`item.callspec.params` is always a `Dict[str, Any]` ?

Reply:
if so, then all good, if not, then we are probably missing some test cases

Reply:
Yes! `item.callspec.params` is a dictionary that contains all parameter objects. You can see here in the pytest docs:

https://docs.pytest.org/en/6.2.x/reference.html?highlight=callspec#function 
> If given, this is function has been parametrized and the callspec contains meta information about the parametrization.